sparse-11/usr/bin/droid/droid-bootctl.sh

```diff
@@ -0,0 +1,14 @@
+#!/bin/sh
+
+# Check currentl used boot slot
+SLOT="$(/usr/libexec/droid-hybris/system/bin/bootctl get-current-slot)"
+
+# Check if current slot is marked successful
+/usr/libexec/droid-hybris/system/bin/bootctl is-slot-marked-successful $SLOT 2> /dev/null
```

Comment:
Why do we need to check this before marking as successful?

Reply:
Not sure why it was done like that, probably to stop it from always setting it if it's already ok.